modules/contributing-quick-starts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="contributing-quick-starts_{context}"]
+= Contributing quick starts
+
+In {product-title} 4.7, a quick start custom resource is added. This allows Operators and administrators to contribute new quick starts to the cluster beyond the out-of-the-box set. Typically, quick starts for Operators are created by the Operator itself after the Operator is installed. In a few cases, out-of-the-box quick starts that guide administrators through the process of installing an operator are available. These quick starts must be created before Operator installation through Operator Hub.
+
+.Procedure
+
+When the quick start is complete and the content guidelines are applied, contribute your quick start to the `quickstarts` folder in the link:https://github.com/openshift/console-operator/tree/master/quickstarts[console-operator] repository.
+
+. Add the quick start YAML to the `quickstarts` folder. Quick start resources in the `console-operator` repository must contain the following cluster profile annotations:
++
+[source,yaml]
+----
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+----
```

Comment:
@ahardin-rh these annotations are due to availability and should be on all the resources that are created by any operator's manifests. 
As you can see above these created operators will be contributed to the console-operator repo, into the `quickstarts/` folder. When the console-operator is being built, it will [copy](https://github.com/openshift/console-operator/blob/master/Dockerfile.rhel7#L16) them into it's manifests, and they will be automatically created upon the cluster installation.

Reply:
These annotations are only needed for Red Hat quick starts that ship with the product as part of the console operator. A user wouldn't need to add them to their own quick starts.